picard/util/tags.py

```diff
@@ -98,6 +152,31 @@ def script_name(self):
         else:
             return self._name
 
+    def notes(self):
+        for attrib, note in ATTRIB2NOTE.items():
+            if getattr(self, attrib):
+                yield note
+
+    def settings(self):
```

Comment:
Perhaps it would be clearer if we used `related_options_titles`, `settings` is used elsewhere already and it can be misleading.

Reply:
Good point.  I'll change it to `related_options_titles` as you suggested.